### ISSUE TITLE
Issues loading some pages with older openGl

### ISSUE THREAD

**Issue by TJTorola** [qutebrowser/qutebrowser#7144]
**Version info**:
```
qutebrowser v2.5.0
Git commit: 
Backend: QtWebEngine 5.15.9, based on Chromium 87.0.4280.144
Qt: 5.15.3 (compiled 5.15.2)

CPython: 3.10.4
PyQt: 5.15.6

sip: 6.6.1
colorama: no
jinja2: 3.0.3
pygments: no
yaml: 6.0
adblock: no
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.5
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.38.2
QtNetwork SSL: OpenSSL 1.1.1n  15 Mar 2022

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Mesa/X.org, 2.1 Mesa 22.0.2
Platform: Linux-5.17.1-1-aarch64-ARCH-aarch64-with-glibc2.35, 64bit
Linux distribution: Arch Linux ARM (arch)
Frozen: False
Imported from /usr/lib/python3.10/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/tjtorola/.cache/qutebrowser
config: /home/tjtorola/.config/qutebrowser
data: /home/tjtorola/.local/share/qutebrowser
runtime: /run/user/1001/qutebrowser
system data: /usr/share/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:00:04
```

**Description**
I'm running linux under qemu on a new m1 mac. Due to some issues with this setup it is limited to OpenGl 3.0 or less. Looking at the :version info it seems like qutebrowser is using 2.1 (though glxinfo shows 3.0 is available).

The issue is some pages get stuck loading. One example is the github.com homepage (when you aren't logged in) will load up until the fancy 3d connected globe animation they have starts to load. At that point qutebrowser gets stuck and becomes unresponsive.

On this same setup I can browse those pages on other browsers like chromium and vieb.

**Steps to repro**
_With a system that only has opengl <=3.0_
- Go to github.com

**Comment by The-Compiler** [comment]
It's unlikely there is anything qutebrowser can do about this.

Could you please try another browser using QtWebEngine such as KDE Falkon?

**Comment by TJTorola** [comment]
Yes, I just did that, the same issue exists there. Is there an issue tracker for Qt that I can report this too?

**Comment by TJTorola** [comment]
Well messing around, I just did `:set opengl.enabled false` and that fixed it. Oddly, the 3d animation on github.com started working after I disabled it. (maybe they have some kinda video backup?).

**Comment by TJTorola** [comment]
With how similar this is to the experience in chromium and vieb, I'm now guessing that those browsers automatically disabled opengl with the version I have.

**Comment by The-Compiler** [comment]
I'm confused - when you say opengl, do you mean WebGL, and when you say `opengl.enabled`, do you mean `content.webgl`?
